### DISCUSSION TITLE
Nextra's Layout with [...slugs].tsx?

### DISCUSSION THREAD

**Discussion by shawn-ycx** [shuding/nextra#1242] [Q&A]
👋🏻 
Is there a way for dynamic markdowns pulled in from a CMS with dynamic routes to work with Nextra's Layout?

If there are existing resources related to this, can someone kindly point me in the right directions?

**Comment by dimaMachina** [comment]
this will be part of v3 but with mdx extension `[...slugs].mdx`

**Reply by ChiaYuSu** [reply]
is there any reference now, thank you

**Comment by dimaMachina** [comment]
https://the-guild.dev/blog/nextra-3#remote-docs-support

**Comment by WBBB0730** [comment]
Hello! Have you found a solution yet? I’m encountering the same issue: I’m currently using Nextra v2 and, for certain reasons, I can’t upgrade to v3. However, I really need dynamic routing to load some articles. If you’ve found a workaround, I would be incredibly grateful if you could share it with me.
